src/components/common/template/TemplateTitle.js

```diff
@@ -27,31 +26,36 @@ export default function TemplateTitle(props) {
       // setSelectValue(selectValue.concat({ _id: showValue._id }));
       setContent(showValue.map((obj) => obj["content"]).join("\n"));
     } else {
-      setSelectValue({ _id: showValue[0]._id });
+      setSelectValue({ _id: showValue[0].id });
       setContent(showValue[0].content);
     }
+    handleClose();
   };
+
   const handleClose = () => setModalValue(false);
 
   return (
-    <box>
+    <Box
+      sx={{
```

Comment:
👍 But it would have been better to use styled components according to our convention.😢